Delete.java

```diff
@@ -0,0 +1,31 @@
+
+import java.io.File;
+
+/**
+ * class Delete that has a boolean method deleteFile
+ *
+ * @author Milena Vallejos
+ * @date May 19
+ */
+public class Delete {
+	/**
+	 * Method to deleteFile from a filePath
+	 *
+	 * @param filePathToDelete is a String (file path)

```

Comment:
is this an absolute or relative path? does this path consider the name of the file? and it does the same for directories? you need to put all that information in the documentation